modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3752,6 +3752,8 @@ Use sliding window compaction.
 
 === log_compression_type
 
+IMPORTANT: Only the `producer` value has any effect. Redpanda brokers do not compress or recompress data based on this property. If producers send compressed data, Redpanda stores it as-is; if producers send uncompressed data, Redpanda stores it uncompressed. Other listed values are accepted for Apache Kafka compatibility but are ignored by the broker. This property may appear in Admin API and rpk topic describe outputs for compatibility.
```

Comment:
> Only the `producer` value has any effect

Really no matter what value is used we ignore it and the behavior is the same as `producer`.